internal/tiltfile/config/config_test.go

```diff
@@ -205,7 +205,7 @@ cfg = config.parse()
 
 	_, err := f.ExecFile("Tiltfile")
 	require.Error(t, err)
-	require.Equal(t, "positional args were specified, but none were expected (no arg defined with args=True)", err.Error())
+	require.Equal(t, "positional args were specified, but none were expected (no flag defined with args=True)", err.Error())
```

Comment:
...hm where would `args=True` be specified? I'm confused.

Reply:
This is an argument to `config.define_string_list`, e.g. https://github.com/windmilleng/tilt.build/pull/325/files#diff-0f16fdfc562a71a859b08af2266710ffR33